common/Field_Dimensions.hpp

```diff
@@ -56,6 +57,9 @@ struct Field_Dimensions {
     inline Geometry2d::Rect OurHalf() const { return _OurHalf; }
     inline Geometry2d::Rect TheirHalf() const { return _TheirHalf; }
 
+    //We play on a 2D rectangular field, hence the length "4" of the array
+    inline std::array<Geometry2d::Line, 4> FieldBorders() const { return _FieldBorders; }
```

Comment:
If you change this to return a reference, it will be slightly faster because it won't have to copy the field border array.  This would look like:

~~~{.cpp}
inline std::array<Geometry2d::Line, 4>& FieldBorders() const { return _FieldBorders; }
~~~